ntpcheck/cmd/utils.go

```diff
@@ -170,13 +170,50 @@ func printLeap() error {
 	return nil
 }
 
+// addFakeSecondZoneInfo reads current zoneinfo db leap seconds and add one in the end of month or year
+func addFakeSecondZoneInfo(srcfile, dstfile string, isMonth int) error {
```

Comment:
looking at the code, `isMonth` should be boolean. It's not C, we have real bools in 2021 :-)

Also, why not just allow passing offset in months to this function? This will allow adding fake second to any month from now, or even to the past if offset is signed.

Reply:
let it be another PR? :)

Reply:
changed to offset in months